.travis.yml

```diff
@@ -15,4 +15,9 @@ before_install:
   - yarn install
 install:
   - tsc -p .
-  - docker-compose up
+  - docker-compose build
+after_success:
+  - if [ "$TRAVIS_BRANCH" == "develop" ]; then 
+    docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"; 
+    docker push musicconnectionmachine/unstructureddata; 
```

Comment:
It does but the tutors would need to set the environment variables ( user name and password ) for Docker hub on travis. I tried linking my github account and docker hub account, to see if I would get the permission to do this, but no luck.
So right now the login fails as the environment variables are not set in travis.